### ISSUE TITLE
Stop ButtonLink wrapping unnecessarily on InfoBanner

### ISSUE THREAD

**Issue by minimalsm** [ethereum/ethereum-org-website#5975]
**Is your feature request related to a problem? Please describe.**

When the paragraph content on our `InfoBanner` component wraps, it also causes the ButtonLink text to wrap unnecessarily.

**Describe the solution you'd like**

When the text wraps onto a second line, the button should not wrap unless it runs out of space.

**Screenshots**

Example in Dutch

<img width="966" alt="Screenshot 2022-04-08 at 12 04 06" src="https://user-images.githubusercontent.com/62268199/162424023-268de11c-a520-4397-a7de-f6f8ef5a9ca2.png">

Example in English

<img width="681" alt="Screenshot 2022-04-08 at 12 08 56" src="https://user-images.githubusercontent.com/62268199/162424500-8a440d89-ae7c-4492-aed1-706beb87ac3f.png">

Desired solution

<img width="702" alt="Screenshot 2022-04-08 at 12 08 48" src="https://user-images.githubusercontent.com/62268199/162424476-340f761a-cd3f-4779-9701-b02c0af61b27.png">

**Additional context**

There are some cases where we DO want this ButtonLink copy to wrap, so removing adding `white-space: nowrap;` in all cases doesn't seem like a viable solution.

**Want to contribute?**

We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.

**Comment by KaranKaira** [comment]
would like to take this on

**Comment by maxehnert** [comment]
> There are some cases where we DO want this ButtonLink copy to wrap

What would an example of that look like?

**Comment by minimalsm** [comment]
@maxehnert one example is with more verbose languages. Ukrainian often requires two lines for the Button copy or it would overflow/cause horizontal scrolling. 

[Here is our homepage in Ukrarian for reference](https://ethereum.org/uk/):

<img width="374" alt="Screenshot 2022-04-12 at 22 41 10" src="https://user-images.githubusercontent.com/62268199/163059190-e15881bc-8fa2-467d-8fbc-c00dc875e383.png">

**Comment by ianrandmckenzie** [comment]
Hi, just poking around the repo. I immediately thought `white-space: nowrap;` as well, so glad you acknowledged it. I would guess it could still work, but maybe adding a `max-width` of some kind could help. It may need responsive queries if going the `max-width` route, though. Just leaving this for whoever is/will work on it. Ping me if needed. :)

**Comment by Dark-Knight11** [comment]
Hi, I would like to work on this issue.
@ianrandmckenzie would this be ok?
button would be wrapped on screen size less than 480px
``` CSS
@media screen and (min-width: 480px) {
  .kQBZHz {
    white-space: nowrap;
  }
}
```

**Comment by ianrandmckenzie** [comment]
@Dark-Knight11 of course, I was just offering a suggestion. Feel free to dive in :)

**Comment by Dark-Knight11** [comment]
I thought of another method to achieve the required result 
instead of being dependant on screen size, we can look for length of text inside the button and apply wrapping accordingly
``` js
var x = document.getElementsByClassName('ButtonLink__StyledLinkButton-sc-a8k023-0');
    for (let i in x) {
        var customStyle = {
            whiteSpace: "nowrap"
        };
        if (x[i].innerHTML.length < 20) {
            Object.assign(x[i].style, customStyle)
        }
    }
```

I'm a little confused about the word limit so can anyone help me with it?

**Comment by Andy-Waine** [comment]
@minimalsm - I know this issue has gone a little stale, but please see the [PR](https://github.com/ethereum/ethereum-org-website/pull/7447) that corrects this. I did end up using `white-space: nowrap;`, but I applied it as inline-styling to ONLY the button mentioned in the issue (in 17 different languages), this way no changes will occur to any other ButtonLink tags. Thanks!

**Comment by minimalsm** [comment]
Closed by #7447
